app/src/main/cpp/skyline/vfs/ticket.h

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <vfs/backing.h>
+#include <crypto/key_store.h>
+
+namespace skyline::vfs {
+    struct Ticket {
```

Comment:
There should be a comment here, at the *very least* linking to Switchbrew but I'd have a `@brief` comment explaining what tickets are and their function on a basic level

Reply:
How does this look
```
/**
  * @brief The Ticket struct allows easy access to ticket files, a format used to store an encrypted title keys
  * @url https://switchbrew.org/wiki/Ticket
  */
```

Reply:
LGTM!